src/System.Memory/tests/ParsersAndFormatters/SupportedFormats.cs

```diff
@@ -69,8 +69,8 @@ public static IEnumerable<SupportedFormat> FloatingPointFormats
         {
             get
             {
-                yield return new SupportedFormat('G', supportsPrecision: false) { IsDefault = true };
-                yield return new SupportedFormat('g', supportsPrecision: false) { FormatSynonymFor = 'G', ParseSynonymFor = 'G' };
+                yield return new SupportedFormat('G', supportsPrecision: true) { IsDefault = true };
```

Comment:
This just enables the existing logic to do the precision checks.

In the 3.0 branch, this was all replaced with more explicit tests that are shared between x86 and x64 instead.